### ISSUE TITLE
Phone verification error occurs

### ISSUE THREAD

**Issue by moonsubae** [vysheng/tg#1732]
phone number: +821012341234
 *** 1638365853.511603 Notification API_64BIT_LOGIN_APP_OUTDATED_153: You are using an outdated app that is no longer supported. To access your messages, please update your app to the latest version.
 *** 1638365853.512399 Incorrect phone number

**Comment by bbyfacekiller** [comment]
Telegram changed something today. I don't know if anyone it's going to modify this CLI tool.

**Comment by v4rchaem** [comment]
telegram cli stopped working, help to solve the issue.

**Comment by goro22052** [comment]
I have the same problem...

**Comment by Paulchen232** [comment]
The  problem is from telegram solve. you do the new login with the parm `-q`.

**Comment by goro22052** [comment]
I try to run command:
telegram-cli -q
but I get 
"
*** 1641732944.969012 Notification API_64BIT_LOGIN_APP_OUTDATED_36: You are using an outdated app that is no longer supported. To access your messages, please update your app to the latest version. If you don’t know how to update your app, go to https://telegram.org/dl
 *** 1641732944.969690 incorrect code
"

**Comment by Paulchen232** [comment]
For me it works on Raspberry Pi Zero. 

/home/pi/tg/bin/telegram-cli  -W -s /home/pi/tg/test.lua -k /home/pi/tg/tg-server.pub
Telegram-cli version 1.4.1, Copyright (C) 2013-2015 Vitaly Valtman
Telegram-cli comes with ABSOLUTELY NO WARRANTY; for details type `show_license'.
This is free software, and you are welcome to redistribute it
under certain conditions; type `show_license' for details.
Telegram-cli uses libtgl version 2.1.0
Telegram-cli includes software developed by the OpenSSL Project
for use in the OpenSSL Toolkit. (http://www.openssl.org/)
I: config dir=[/home/pi/.telegram-cli]
HI, this is lua script

**Comment by Paulchen232** [comment]
@goro22052 How you have it installed?

**Comment by goro22052** [comment]
apt install telegram-cli

**Comment by Paulchen232** [comment]
Ok, look her:
[https://github.com/vysheng/tg](https://github.com/vysheng/tg)
how to install it too.

**Comment by goro22052** [comment]
I had success verification from a first phone number, which has telegram account in the past, 
but I can't verification from my second number, which hasn't telegram account in the past, ...

**Comment by Paulchen232** [comment]
I don't know what you with "asn't telegram Konto"means.

**Comment by goro22052** [comment]
for my second phone number, it's the first time when I tried entered into telegram

**Comment by Paulchen232** [comment]
Ok, then try it in [web.telegram.org](web.telegram.org). Perhaps you get more Infos.

**Comment by TBR-BRD** [comment]
> telegram-cli -q

i use the same, no longer...

any ideas?

**Comment by Paulchen232** [comment]
@TBR-BRD:
What you me with "no longer..."
If you install it with GitHub then the error "API_64BIT_LOGIN_APP_OUTDATED_153" comes not. @goro22052 have say "

**Comment by Paulchen232** [comment]
"I had success verification"

**Comment by burkass** [comment]
yes just try it and I go the error API_64BIT_LOGIN_APP_OUTDATED after entering the code. It's happen since yesterday.

![image](https://user-images.githubusercontent.com/5495419/152323332-6f0b4736-70a4-4411-9d70-e65df29c3105.png)

**Comment by Paulchen232** [comment]
How you have it installed?

**Comment by Paulchen232** [comment]
And what Linux you use? I don't know why it at my works. Long ago I have with `-q` new login after the error.

**Comment by burkass** [comment]
> And what Linux you use? I don't know why it at my works. Long ago I have with `-q` new login after the error.
Yes already but still failed. but for existing that activated 1 month back, send message still normal. Only occur when activation.

Anyway sorry late response

![image](https://user-images.githubusercontent.com/5495419/152668942-1f846b92-27ad-4953-a7a7-ee23181054ae.png)

**Comment by v8w1** [comment]
I have the same problem...

![image](https://user-images.githubusercontent.com/18752348/190457164-74948a80-3cf2-4275-b461-e70cebd58f88.png)

**Comment by LexyJexon** [comment]
Same problem, please help!!!

**Comment by FedericoBruzzone** [comment]
I have the same issue even when using -q :(

**Comment by kalia1962** [comment]
I've got the same error on Manjaro 23 after some updates (not sure whoch exactly) - asks for phone number and then:

 *** 1686658986.428622 Notification API_64BIT_LOGIN_APP_OUTDATED_186: You are using an outdated app that is no longer supported. To access your messages, please update your app to the latest version. If you don’t know how to update your app, go to https://telegram.org/dl
 *** 1686658986.428672 Incorrect phone number

At the same time it works well withought any problems on my Kubuntu 20.04 with all updates.
Maybe it can be made as some kind of "portable" app, Appimage or anything like it, with statically linked libs?
Is it possible?
Or maybe run it in Docker?
